tests/int/test-git_mirror.bsh

```diff
@@ -374,5 +420,31 @@ begin_archive_test "Archive prep_dir"
   begin_fail_zone
   [ "$(tar ztf "${PREP_FILE[0]}")" = "${ans}" ]
   end_fail_zone
+
+  # Test incremental archive (if possible)
+  if [ -f "${PREP_FILE[0]%.tgz}.snar" ]; then
+    touch "${ARCHIVE_DIR}"/newfile
+
+    sleep 2 # the archives are only datetime stamped out to seconds
+    archive_mirrors "${ARCHIVE_DIR}"
+
+    INCREMENTAL_PREP_FILE="$(shopt -s nullglob; echo "${ARCHIVE_DIR}"/transfer_*_transfer_*.tgz)"
+    ans=$'./\n./subdir/\n./newfile' # BSD tar does not appear to support incremental backups
+    [ "$(tar ztf "${INCREMENTAL_PREP_FILE}")" = "${ans}" ]
+
+
+    # Test deleting the last (incremental) backup and re-generating it
+    # Must delete the full and incremental archives and the snar files
+    last_snar_file="$(ls "${ARCHIVE_DIR}"/transfer_*.snar 2>/dev/null | tail -n1)"
+    rm "${last_snar_file}" "${last_snar_file%.snar}.tgz" \
+        "${INCREMENTAL_PREP_FILE}" "${PREP_FILE%.tgz}.l1.snar"
+
+    sleep 2 # the archives are only datetime stamped out to seconds
```

Comment:
You can use `touch` to change the timestamp to anytime you want

Reply:
The archives are datestamped. But I can rename the old archive